### PR TITLE
CDS-13 Add services card to services page

### PR DESCRIPTION
**Pull request by Liannus**
Changelog:
- Add services card to services page.

### CODE REVIEW

File: src/components/Service/Service.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@import "styles/mediaQueries.scss";
+
+#service {
+  #service-image {
+    height: 500px;
+    @include sm {
+      height: 300px;
+    }
+    @include md {
+      height: 400px;
```
Comment (forkball) [inline src/components/Service/Service.scss:11]: Should be done with tailwind.

File: src/components/Service/Service.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@import "styles/mixins.scss";
+@import "styles/mediaQueries.scss";
+
+#service {
+  #service-image {
+    height: 500px;
```
Comment (forkball) [inline src/components/Service/Service.scss:6]: Should be done with tailwind.

File: src/components/Service/Service.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    @include sm {
+      height: 300px;
```
Comment (forkball) [inline src/components/Service/Service.scss:8]: Tailwind it up.

**Review by forkball — CHANGES_REQUESTED** [review]
Some changes to tailwind, otherwise good stuff.

File: src/components/Service/Service.scss (hunk 1 above)
Comment (Liannus) [reply src/components/Service/Service.scss:11]: Tailwind unfortunately caps out at 256 pixels as far as I know.

File: src/components/Service/Service.scss (hunk 2 above)
Comment (Liannus) [reply src/components/Service/Service.scss:6]: See above

File: src/components/Service/Service.scss (hunk 3 above)
Comment (Liannus) [reply src/components/Service/Service.scss:8]: See above

File: src/components/Service/Service.scss (hunk 1 above)
Comment (forkball) [reply src/components/Service/Service.scss:11]: If that's the case, do they really need to be that large?

Comment (Liannus) [reply src/components/Service/Service.scss:11]: Take a look, I believe they do.

Definitely a downside to tailwind.

Comment (forkball) [reply src/components/Service/Service.scss:11]: Alright, we can settle then, not really a big deal. I checked em out and they looked good, we may want to do this to service images on landing.

**Review by forkball — APPROVED** [review]

**Pull request merged**